blocks/api/post.pegjs

```diff
@@ -91,11 +140,11 @@ Special_Chars
 Newline
   = [\r\n]
 
+Whitespace
+  = [\s\r\n]+
```

Comment:
Is `\r` and `\n` somehow not included in `\s` for pegjs? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp#character-classes